{{:Team:CSMU_NCHU_Taiwan/Templates/Header}}
<html>


<html lang="en">
  <head>
    <meta charset="UTF-8">
    <link href='https://fonts.googleapis.com/css?family=Arizonia' rel='stylesheet'>
    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet'>
    <title>software</title>
  </head>

<style type="text/css">

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box; }

body {
  width: 100%;
  margin: 0;
  padding: 0;
  font-family: 'Ubuntu' !important; }
img.center {
  display: block;
  margin: 0 auto; }


img.right {
    float: right;
  }

.bg-container {
  overflow: hidden;
  position: relative;
  background-color: #c5d7d9;
  background-image: url(http://2017.igem.org/wiki/images/8/8d/Csmuxnchu_result_bg_2.png);
  background-repeat: repeat-y;
  background-size: 100%;
}

.my-main-container {
  width: 100%;
  padding: 50px 7%;
 }

.main-content {
  background-color: white;
  width: 80%;
  margin-left: 10%;
  padding: 50px;
  min-height: 180vh;
  z-index: 10; }

.m-text-area h1 {
    font-size: 60px;
    font-weight: 700;
    font-family: 'Arizonia' !important;
    text-align: center;
    margin-bottom: 1rem; }
.m-text-area p {
    font-size: 22px !important;
    font-family: 'Ubuntu' !important;
    font-weight: 500; }

.m-block img {
  width: 100%; }

.path-btns {
  position: fixed;
  width: 250px;
  top: 200px;
  left: 225px; }
  .path-btns .path-dot {
    width: 25px;
    height: 25px;
    border-radius: 50%;
    position: absolute;
    background-color: #fff;
    top: 5px;
    left: 27px;
    border: 5px solid #385e66;
    cursor: pointer; }
    .path-btns .path-dot.active {
      background-color: #385e66; }
    .path-btns .path-dot:hover {
      transition: all .3s;
      transform: scale(1.2); }


.path {
  height: 65px;
  display: flex; }
  .path .pathSvg {
    display: block; }
  .path .pathWord {
    padding-right: 10%; }
  .path .path-btn {
    cursor: pointer;
    fill: #fff; }
    .path .path-btn.path-active {
      fill: #385e66; }
  .path .path-word-sm {
    font-size: 12px; }
#HQ_page .path .pathWord p {
      font-weight: 700;
      text-align: left !important;
      font-size: 16px;
}
#HQ_page .text-area p {
  font-size: 22px;
  font-family: 'Ubuntu';
  font-weight: 500; }
img.center {
  display: block;
  margin: 0 auto; }

 .text-area h2 {
    text-align: center;
    margin-bottom: 2rem !important;
    margin-top: 2rem;
    color: black !important; }

.m-text-area {
  padding-left: 225px; }
  .m-text-area h2 {
    text-align: center;
    margin-bottom: 2rem !important;
    margin-top: 2rem;
    color: black !important; }

.path-tags{
  position: fixed;
  left: 15%;
  font-size: 22px;
  top: 200px;
}
.path-tags ul{
  color: #385e66;
  list-style: none;
font-weight:600;
  font-size: 16px;
}

.path-tags li{
  cursor: pointer;
font-size:16px;
}

.path-tags span{
    color: #fff;
    font-weight:600;
    font-size:22px;
    position: relative;
    top: -8px;
    left: 10px;
    z-index: 1;

    font-size: 15px;
}
.top {
  position: fixed;
  right: 50px;
  bottom: 50px;
  height: 50px;
  width: 50px;
  cursor: pointer; }
  .top img {
    width: 100%; }
.m-text-area h3 {
    font-size: 28px;
    text-align: left;
    font-weight:700;
    margin-bottom: 2rem !important;
    margin-top: 1rem;
    color: #385e66 !important; }
.m-text-area h2 {
    text-align: center;
    margin-bottom: 2rem !important;
    margin-top: 2rem;
    color: black !important;
    font-size:35px;
    font-weight:bold;
     }


.img-container {
  width: 100%;
  text-align: center;
  display: none; }
  .img-container img {
    width: 100%; }


.pdfbtn {
  width: 100%;
  margin: 20px 0 10px 0;
  padding: 10px 20px;
  border-radius: 5px;
  background-color: #102E4A;
  color: white;
  display: block;
  position: relative; }
  .pdfbtn i {
    position: absolute;
    right: 50px;
    top: 50%;
    transform: translateY(-50%); }
.ul-style li{

 font-size:22px;
list-style-type:none;
list-style-image:url(' http://2017.igem.org/wiki/images/f/f1/Csmuxnchu_checkcheck.png ');


}


.ul-container {
  width: 100%;
  text-align: center;
  height: auto;
  display: none; }

.ul-container ul.medal li {

  list-style: none; }
  .ul-container ul.medal li:before {
    content: '';
    display: inline-block;
    height: 1em;
    width: 2em;
    background-size: 1em;
    background-image: url(../img/circle.png);
    background-repeat: no-repeat; }

.ul-container ul {
  text-align: left;
  font-family: 'Ubuntu';
  padding: 20px 20px;
  margin-top: -20px;
  margin-left: 0;
  border: #556b7e solid 2px;
  font-size: 22px; }
  .ul-container ul.ul-brown {

    border: #102E4A solid 2px; }
  .ul-container ul li {
margin-left: 30px;
margin-top: 50px;
margin-bottom: 20px;
    padding: 10px;
    list-style: disc; }
    .ul-container ul li a {
      text-decoration: none;
      color: #000; }
      .ul-container ul li a:visited, .ul-container ul li a:hover, .ul-container ul li a:focus {
        color: #000;
        text-decoration: none; }
    .ul-container ul li span {
      background-color: #fadd3a;
      padding: 1px 10px;
      border-radius: 5px;
      margin-left: 20px;
      cursor: pointer; }
    .ul-container ul li i {
      padding-left: 10px; }

.ul-style-num ul2 li{
list-style-type:decimal;
letter-spacing: 3;

}
</style>

<body>
    <div class="bg-container" style="max-height:none;">
<a name="antidote"></a>
      <div class="my-main-container">
        <div class="main-content">
          <div class="m-text-area">
            <h1>Results</h1>

            <div id="model-intro" class="m-block" >

                <img class = "center" id="antidote" src="http://2017.igem.org/wiki/images/1/16/T--CSMU_NCHU_Taiwan--design-antidote.png" alt="" style="width:40%; margin-top:20px;">
                 <h2 id="expression">Expression results</h2>
                 <ul class="ul-style">
                   <li>We used IPTG to induce two of our composite parts and two plasmids from Australian team in <l>E. coil</l> and extracted the cell lysates.</li>
                   <li>We found two types of MSMEG5998 had great solubility.</li>
                    <li>We purified four target proteins by Nickel-resin affinity chromatography.</li>
                 </ul>
                 <div class="pdf-area">
                      <span class="pdfbtn" id="result-btn-1">IPTG induction<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="result-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;Two of the composite parts (BBa_K2382005 and BBa_K2382006) were synthesized by Allbio Life Co., Ltd and put into the standard backbone pSB1C3. First, we transformed all of our plasmids, including two plasmids from Australia and two of our own composition parts into E. coli BL21 (DE3) strain to express our proteins. Then IPTG was used to induce the expression system since all plasmids in our project had T7 promoter. We sonicated E. coli and did 9500 rpm and 13000 rpm centrifugation to remove the cell pellet and obtain the supernatant. To confirm the suitable concentration of cell supernatant, we did SDS-PAGE electrophoresis and coomassie brilliant blue staining. The results are demonstrated in the <strong>Fig. 1A </strong>to <strong>1C</strong>. After centrifuging two times, we could find a high percentage of proteins in the cell supernatant (the 13000 Su group).</p>                <br>
                           <img src="http://2017.igem.org/wiki/images/9/95/Csmuxnchu_result1.png" style="width:100%">
                           <p><small><small><strong>Fig. 1:</strong>Cell lysates were analyzed by SDS-PAGE and coomassie brilliant blue staining. 9500 T meant the initial sample obtained after sonication; 9500 P and 13000 T meant the pellet and the supernatant gotten after 9500 rpm for 20 min; 13000 P and 13000 Su meant the pellet and the supernatant obtained after 13000 rpm for 20 min. (A)(B) Samples contained Australian and synthetic MSMEG5998. (C) Samples contained Australian and synthetic FGD.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>

                  <div class="pdf-area">
                      <span class="pdfbtn" id="result-btn-2">Protein purification, and dialysis<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="result-2">
                         <ul class="ul-brown">
                          <div class="text-area">
                           <p>&emsp;After extracting the cell lysates, we used nickel-resin column to purify our target proteins from the cell lysates because all of our proteins were tagged with 6 histidines at their C-terminal ends. After protein purification,and protein dialysis with dialysis buffer containing 150 mM NaCl, 20 mM Tris-HCl (pH=7.5), and 20% glycerol to remove imidazole in our purified proteins, we did SDS-PAGE gel electrophoresis to ensure our target proteins were successfully purified <strong>(Fig. 2A and 2B)</strong>. The molecular weights of these proteins are listed in the<strong> Table 1</strong>. The standard BSA proteins were used to quantify the concentration of target proteins.</p><br>
                             <img src="http://2017.igem.org/wiki/images/4/43/Csmuxnchu_result2_1.png" style="width: 100%">
                              <p><small><small><strong>Table 1:</strong>Four expressed recombinant proteins and their molecular weights are listed.</small></small></p><br>
                               <img src="http://2017.igem.org/wiki/images/5/51/Csmuxnchu_result2_2.png" style="width:100%">
                               <img src="http://2017.igem.org/wiki/images/4/4c/Csmuxnchu_result2_3.png" style="width:100%">
                               <p><small><small><strong>Fig. 2:</strong>Concentration of proteins was quantified by SDS-PAGE and standard BSA samples with 0.16, 0.31, 0.63, 1.25, 2.5, 5, 10 mg/ml. (A) Two recombinant proteins were expressed by the plasmids from Australia. (B) Two recombinant proteins expressed by the plasmids from DNA synthesis.</small></small></p><br>


                           </div>
                         </ul>
                       </div>
                 </div>
                  <div class="pdf-area">
                      <span class="pdfbtn" id="result-btn-3">Protein solubility analysis<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="result-3">
                         <ul class="ul-brown">
                          <div class="text-area">
                                <p>&emsp;To know whether the solubility of our two enzymes increased after fusing enzymes with thioredoxin, we dissolved all cell lysates which containing pellet and supernatant and did western blot to detect the content of our target proteins. All proteins were detected by anti-6x His Tag antibody because all of them contained a 6-histidines tail when bacteria expressed them. In<strong> Fig. 3</strong>, we could find there was good expression of both Australian and synthetic MSMEG5998 in the “13000 Su” group when compared with the “13000 P” group. This result meant that most proteins were dissolved in the supernatant while few proteins deposited in the cell pellet after 13000-rpm centrifugation. However, we could not observe good solubility in both Australian and synthetic FGD because there were little or no difference between the “13000 Su” group and the “13000 P” group.</p><br>
                               <img src="http://2017.igem.org/wiki/images/b/b6/Csmuxnchu_result3.png" style="width:100%">

                                <p><small><small><strong>Fig. 3:</strong>Cell lysates in the process of two times centrifuge were analyzed by western blot. The abbreviations of five groups were the same as Fig. 1.</small></small></p><br>
                           </div>
                         </ul>
                      </div>
                 </div>

            </div>
            <div id="model-protein" class="m-block" >
                <h2 class="m-subtitle" id="aflatoxin">Aflatoxin damage results</h2>
                 <ul class="ul-style">
                   <li>We found aflatoxin B1 didn't affect the cell viability too much.</li>
                   <li>We found aflatoxin B1 led to DNA damage in hepatoma cell line obviously at 48 hour.</li>
                    <li>We found aflatoxin B1 activated the p53 pathway but not the apoptosis pathway.</li>
                 </ul>

                 <div class="pdf-area">
                      <span class="pdfbtn" id="damage-btn-1">Cell viability assay<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="damage-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;Since very often aflatoxin B1 leads to mutation and is followed by oncogenesis in the liver, we had to decide on a suitable concentration where aflatoxin would cause DNA damage rather than cell death. Therefore, we studied the cell viability first. HepG2 (human hepatic carcinoma, a widely used cell line to observe the aflatoxin-induced DNA damage<sub>[1]</sub>) was treated with aflatoxin B1 or 1% DMSO alone (aflatoxin was dissolved in DMSO) for 24 and 48 h and we analyzed the cell viability (<strong>Fig.4</strong>). Compared to the untreated cells, cell death of 10% and 20% respectively were observed for cells treated with aflatoxin at 5 and 10 μM in 24 h, but there was no statistical significance. On the other hand, in 48 h, cell death of 32% and 46% respectively were observed at 5 and 10 μM and both of them were significant when compared to the control group (p < 0.001). At the same time, there was no significant difference between DMSO-treated group and the control group in 24 h and 48 h. This result indicated that the solvent of aflatoxin did not affect the cell viability.</p>                <br>
                           <img src="http://2017.igem.org/wiki/images/2/20/Csmuxnchu_result_4.png" style="width:100%"><br>
                           <p><small><small><strong>Fig. 4:</strong>Effect of aflatoxin on cell viability in HepG2 cells. HepG2 cells (2×104/well in 96-well plate) were treated with 0, 1.25, 2.5, 5, 10 μM aflatoxin B1 or 1% DMSO alone for 24 and 48 h. Cell viability was analyzed by using MTT assay. The data is presented with means ± S.E. and analyzed using one-way ANOVA post-Hoc Tukey’s test. ***, p < 0.001 compared with the control group.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>
                 <div class="pdf-area">
                      <span class="pdfbtn" id="damage-btn-2">Aflatoxin led to DNA damage in HepG2<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="damage-2">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;In previous studies, H2AX phosphorylation is a key step in the DNA damage response (DDR), playing a role in signaling and initiating the repair of double strand breaks (DSBs)<sub>[2]</sub>. To figure out whether aflatoxin B1 induced DNA damage in HepG2 cells, we did immunocytochemistry after 48h treatment to detect the expression of γH2AX by anti-γH2AX antibody. In <strong>Fig. 5</strong>, we could observe some obvious fluorescence signals when treatment of aflatoxin B1 10 μM. Simultaneously, we only found few signals in the treatment of 2.5 and 5 μM. There was no difference between the vehicle group (1% DMSO) and the control group, so this group could be the negative control.</p>                <br>
                           <img src="http://2017.igem.org/wiki/images/d/d8/Csmuxnchu_result_4_2.png" style="width:100%">
                           <p><small><small><strong>Fig. 5:</strong>Effect of aflatoxin on DNA damage in HepG2 cells. Cells (1×105/well in 24-well plate) were treated with 0, 2.5, 5, 10 μM aflatoxin B1 or 1% DMSO alone for 48 h.γH2AX was analyzed by immunocytochemistry.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>
               <div class="pdf-area">
                      <span class="pdfbtn" id="damage-btn-3">Aflatoxin-induced p53 pathway activation<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="damage-3">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;From previous studies, p53 was induced by human Chk1/Chk2 when cells underwent DNA damage and which will then activate some proteins like p21 to cause cell cycle arrest<sub>[3, 4]</sub>. Accordingly, we checked some protein expression in p53 pathway after treating aflatoxin B1. HepG2 cells were treated with four concentrations of aflatoxin or 1% DMSO alone for 24 h. Afterwards the cells were lysed with RIPA buffer containing protease inhibitor and phosphatase inhibitor. Protein concentration was assayed with Bio-Rad Protein Assay Kit. Equal amounts of proteins from each sample were analyzed using western blot. In <strong>Fig. 6A</strong>, p-Chk1 (Ser345), p53, p21 were induced by aflatoxin in HepG2 cells but only the expression of p21 was obviously increased in dose-dependent manner. At the same time, we also checked the phosphorylation sites of p53 and found only the site of Serine20 was distinctly observed (<strong>Fig. 6B</strong>). It may indicate that the kinase, p-Chk1, phosphorylated p53 prevented it from being degraded by mdm2<sub>[5]</sub>. In addition, to make sure aflatoxin only causes DNA damage rather than apoptosis in short-term treatment, we also checked some apoptosis pathway markers. As of our hypothesis, there were no significant difference between aflatoxin groups and the untreated group (<strong>Fig. 6C</strong>). For all western blotting results in <strong>Fig. 6</strong>, they resemble protein expressions between the control group and the DMSO group, which indicated that the solvent of aflatoxin did not interfere with the results. </p>                <br>
                           <img src="http://2017.igem.org/wiki/images/1/1b/Csmuxnchu_result_5_1.png" style="width:100%">
                            <img src="http://2017.igem.org/wiki/images/f/fd/Csmuxnchu_result_5_2.png" style="width:100%">
                         <img src="http://2017.igem.org/wiki/images/b/b1/Csmuxnchu_result_5_3.png" style="width:100%">
                           <p><small><small><strong>Fig. 6:</strong>Induction of p53 pathway rather than apoptosis pathway in HepG2 cells by aflatoxin treatment (0, 2.5, 5, 10 μM and another group of 1% DMSO). All data were standardized with the control group. (A)(C) p-Chk1, p53, p21, PARP, Bax were determined by western blot after HepG2 cells (5×105 cells/6 cm dish) were treated with aflatoxin for 24 h. β-actin was a loading control. (B) Different phosphorylation sites and total p53 were determined also by western blot after 8×105 cells were treated for 24 h.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>


            </div>
            <a name="enzyme_function_results"></a>
            <div id="model-docking" class="m-block" >
                <h2 class="m-subtitle" id="enzyme">Enzyme function results</h2>
                <ul class="ul-style">
                   <li>We discovered MSMEG5998 had great activity to degrade aflatoxin B1 in vitro,and the synthetic one was even better then the original one.</li>
                   <li>We found MSMEG5998 alleviated aflatoxin-induced activation of p53 pathway.</li>
                </ul>
                <div class="pdf-area">
                      <span class="pdfbtn" id="function-btn-1">Enzyme activity assay<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="function-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;The conditions of reaction to degrade aflatoxin by MSMEG5998 were modified from Taylor’s study<sub>[6]</sub>. All concentrations of reactants are listed in Table 2 and 32 μM aflatoxin was used. We first mixed all reactants in eppendorfs and then put them at 22°C.</p><br>
                          <p>&emsp;In <strong>Fig. 7A</strong>, we compared two proteins, MSMEG5998 and F420-dependent glucose-6-phosphate dehydrogenase (FGD) expressed from Taylor’s vectors (from Australia) and from our synthetic vectors. We found that both the Australian and synthetic MSMEG5998 have great activity and degraded aflatoxin B1 by more than 60%. The effect of the synthetic one may be better than the Australian one but there were no statistic significance.</p>

                          <p>&emsp;However, only Australian FGD has activity to reduce F420 into F420H2 and help the reaction. This finding corresponds with our dry lab results. Therefore, we used Australian and synthetic MSMEG5998 and Australian FGD to do the same experiment again to figure out whether the degradation percentage was dependent of time and whether the main reason of degradation was MSMEG5998. The results were detected by direct 365 nm absorbance (<strong>Fig. 7B</strong>) and by ELISA (<strong>Fig. 7C</strong>). We found out that the degradation percentage was time-dependent. The synthetic MSMEG5998 had better activity than Australian MSMEG5998. The former was able to degrade 83% aflatoxin after 8 h while the latter could only degrade 52% aflatoxin. </p>
                           <img src="http://2017.igem.org/wiki/images/6/6f/Csmuxnchu_result_6_1.png" style="width:100%">
 <img src="http://2017.igem.org/wiki/images/7/74/Csmuxnchu_result_6_2.png" style="width:100%">
 <img src="http://2017.igem.org/wiki/images/e/e4/Csmuxnchu_result_7_1.png" style="width:100%">
                           <p><small><small><strong>Fig. 7:</strong>MSMEG5998 could significantly degrade aflatoxin at time manner in vitro. <br>
(A) Direct 365 nm absorbance were detected after mixing Australian/synthetic MSMEG5998 and Australian/synthetic FGD and other reactants at 0th and 8th hour in the environment of pH=7.5 and 22℃
.<br>(B) The same way as (A) but Australian/synthetic MSMEG5998 and Australian FGD were used and the reaction were detected at 0th, 2nd, 4th, 6th, and 8th hour. a, p < 0.001 compared to the 0th hour of the synthetic MSMEG5998(+) group; b, p < 0.001 compared to the 0th hour of the Australian MSMEG5998(+) group; c, p < 0.001 compared to the same time of the Australian MSMEG5998(+) group.<br>(C) The same way as (B) but the degradation percentage were detected by ELISA. Because the initial concentration of aflatoxin (10000 ng/ml) was too high to be detected by the ELISA, we didn’t demonstrate the initial data.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>
               <div class="pdf-area">
                      <span class="pdfbtn" id="function-btn-2">MSMEG5998 alleviated aflatoxin-induced p53 pathway activation in HepG2<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="function-2">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;Cells (5×105/ 3.5 cm dish) were divided into 5 groups: control group, aflatoxin group (treated with 10 μM), aflatoxin + reactants group (treated with aflatoxin 10 μM and reactants as <strong>Table 2</strong>), aflatoxin + reactants + MSMEG5998 group (treated with aflatoxin 10 μM, reactants in <strong>Table 2</strong> and synthetic MSMEG5998 0.1 μM), and MSMEG5998 group (treated with synthetic MSMEG5998 0.1 μM).</p><br>
                          <p>&emsp;After 24 h treatment, cells were lysed and their protein expression was analyzed by western blot. In <strong>Fig. 8</strong>, we found that the expression of p-Chk1 (Ser345), p-Chk2 (Thr68), p-p53 (Ser20), p53, and p21 were all decreased by MSMEG5998 and other reactants when compared to the aflatoxin alone group. This result may attribute to the highly desirable activity of MSMEG5998 of degrading aflatoxin and preventing the toxin from entering cells. </p><br>
                          <p>Unexpectedly, we also observed a lower expression of these proteins in the aflatoxin + reactants group, which meant that MSMEG5998 may not be the only factor that inhibit the activation of p53 pathway in HepG2 cells. Besides, to ensure that our enzyme MSMEG5998 would not be toxic to cells, we designed a group of MSMEG5998 alone and found that it resembles expression of these proteins as the control group. The results showed that this enzyme was safe.</p><br>

                           <img src="http://2017.igem.org/wiki/images/8/84/Csmuxnchu_result_7_2.png" style="width:100%">
<p><small><small><strong>Table 2:</strong>The substance concentration of the aflatoxin-degradation reaction. For convenience sake, we called G6P, F420, FGD, and tris buffer as the reactants.</small></small></p><br>
<img src="http://2017.igem.org/wiki/images/c/cf/Csmuxnchu_result_8_1.png" style="width:100%">
                           <p><small><small><strong>Fig. 8:</strong>MSMEG5998 decreased the p53 pathway activation induced by aflatoxin in HepG2. p-Chk1, p-Chk2, p-p53 (Ser20), p53, and p21 were detected by western blot after HepG2 cells (5×105 cells/3.5 cm dish) were treated with aflatoxin for 24 h.  β-actin was a loading control. All data were standardized with the control group. C: control group; AF: aflatoxin group; A+R: aflatoxin + reactants group: aflatoxin + reactants + MSMEG5998 group; 5998: MSMEG5998 group.</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>







             </div>
            <div id="model-conslusion" class="m-block" >
                <h2 class="m-subtitle" id="cloning">Cloning results</h2>
                 <ul class="ul-style">
                    <li>We used two restriction enzymes to link the MSMEG5998 gene and pG1-EX1,a shuttle vector for <i>E.coil</i> and yeast ,in order to make yeast carry this gene and express it.</li>
                 </ul>

                <div class="pdf-area">
                      <span class="pdfbtn" id="cloning-btn-1"><font-style:oblique;>E.coil</font> and yeast cloning<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="cloning-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p>&emsp;To insert MSMEG5998 gene into our shuttle vector for E. coli and yeast, pG1-EX1, we first designed two specific primers which contain BamHⅠ (the 5’ end one) and XhoⅠ (the 3’ end one) independently. Second, we did PCR to amplify our gene (the insert) and digested this gene with BamHⅠ and XhoⅠ. At the same time, we digested our plasmid with the same two restriction enzymes to prepare the vector. All results were checked by DNA electrophoresis before ligation. Then we used DNA ligase to ligate the insert and the vector and transformed this recombinant plasmid into <i>E. coli</i> (DH5α). The transformation result was shown in <strong>Fig. 9A</strong>. To check whether the construction was successful, we picked twenty colonies (cultured in another dish and showed in <strong>Fig. 9B</strong>) and extracted their plasmids. Then we did colony PCR to see whether the plasmids in these twenty colonies contained the MSMEG5998 gene. If the E. coli carried the recombinant plasmid, we could observe a clear band in about 700 bp. The DNA electrophoresis results were demonstrated in <strong>Fig. 9C</strong>. We found that most of the colonies carried the recombinant plasmid. Finally, after amplifying our recombinant plasmids in E. coli, we extracted the plasmids and transformed into yeasts to make them express the recombinant protein. There is one gene related to synthesize tryptophan in pG1-EX1, if our recombinant plasmid was successfully transformed into yeast and it could enable yeast to grow on the medium lacking tryptophan. The transform results were demonstrated in <strong>Fig. 9D</strong>.</p>                <br>
                           <img src="http://2017.igem.org/wiki/images/1/14/Csmuxnchu_result_9_1.png" style="width:100%">
<p><b>(C)</b></p>
<img src="http://2017.igem.org/wiki/images/3/34/Csmuxnchu_result_9C.png" style="width:100%">
<p><b>(D)</b></p>
<img src="http://2017.igem.org/wiki/images/2/2a/Csmuxnchu_result_9_3.png" style="width:100%">
                           <p><small><small><strong>Fig 9:</strong>Construction of MSMEG5998 in pG1-EX1. (A) The E. coli (the ligated DNA with MSMEG5998 was transformed in E. coli DH5α) was grown on the LB medium + ampicillin. (B) Twenty colonies from (A) were grown on the LB medium + ampicillin. (C) The MSMEG5998 gene (around 700 bp) were observed by DNA electrophoresis after PCR. (D) The yeast (MSMEG5998 plasmid was transformed in yeast BJ2168) was grown on the YPAD medium lacking tryptophan. “1” to “20”: the number of the colony from (B); “N”: the negative control (around 200 bp).</small></small></p><br>

                      </div>
                   </ul>
                 </div>
                 </div>

            </div>
 <br>
<a name="strip"></a>
           <img src="http://2017.igem.org/wiki/images/a/a8/T--CSMU_NCHU_Taiwan--safety-line.png" style="width:100%;" alt="">
<br>

          <img class = "center" src="http://2017.igem.org/wiki/images/b/b1/T--CSMU_NCHU_Taiwan--design-teststrip.png" alt="" style="width:40%;margin-top:20px;">
           <div id="result-strip" class="m-block" >
            <h2>scFv fusion protein results</h2>
               <div class="pdf-area">
                      <span class="pdfbtn" id="strip-btn-1">Protein expression & Future work<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="strip-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p><strong>Protein expression</strong></p>
                          <p>&emsp;Initially when we were designing the sequence of scFv, our plan was to replace the coloring function of gold nanoparticle with RFP(BBa_K2382010). But the colonies we got after the first transformation did not present visible red colors as we expected. We were not sure whether the problem came from RFP itself, or the function of RFP being affected by other proteins, causing it to fail to show desired results. After the fusion protein purification, we obtained a 57kDa single band that is the same size as our protein, so we were sure that the fusion protein has been produced (<strong>Fig.10</strong>).</p>
                           <p>&emsp;Later we analyzed the binding capacity of the purified protein. In the experiment we first went through dialysis of excess salt and then diluted the samples 2000 and 4000 times before doing ELISA assay. After our test, only 2000 times dilution of the protein seems to perform a bit of affinity (<strong>Fig. 11</strong>). </p>  <br>
                       <p><strong>ELISA</strong></p>
                       <p>&emsp;However, we doubted that the result might be interrupted by salt in buffer. Therefore, we narrowed the dilution range to 2000 times to 4000 times (<strong>Fig. 12</strong>), and desalted before ELISA assay. If antibody have binding ability, the O.D value would change with AFB1 level. Theoretically, the OD we measured would increase with the concentration of aflatoxin. However, the trend line did not follow this rule; the antibody may not perform any binding ability. Unfortunately from the final results we can only judge that the fusion protein does not have any binding ability. </p>
                       <p>&emsp;After subsequent discussion with the instructor, we suspect that the binding ability of scFv is fine, but after the addition of other proteins, the original binding ability may have been change, which led to the loss of the original function.</p><br>

                      <p><strong>Future work</strong></p>
                     <p>We are still managing to find out whether the problem is caused by the scFv sequence itself or the interaction between RFP and scFv.  However, in BioBrick BBa_K2382010, we designed a BamHI restriction enzyme site at the N terminal of EAAAK rigid linker with RFP(BBa_K2382013), giving us a chance to replace another scFv sequence before it. Moreover, we will try to express BBa_K2382010 gene in an expression vector to test whether the EAAAK linker would affect the color of RFP or not.</p>
                     <p>The result above shows that the scFv fusion protein failed to exhibit aflatoxin binding affinity, indicating the construction of a new RFP-carrying scFv that we are going to use in the new immunostrip does not work. However, the kit in our project will not be affected by this result since it is compatible with traditional strips. </p>



                           <img src="http://2017.igem.org/wiki/images/b/bb/Csmuxnchu_result_strip1_1.png" style="width:100%">
                           <p><small><small><strong>Fig. 10:</strong>This result show that bands in 57 kDa were the purified fusion protein. Flow：flow through, Wash：protein washed by wash buffer, E1~E5：Elution1~Elution5</small></small></p><br>
                           <img src="http://2017.igem.org/wiki/images/3/3c/Csmuxnchu_result_fig11.png" style="width:100%">
                           <p><small><small><strong>Fig. 11:</strong>ELISA assay test 2000 times dilution of the scFv fusion protein. If antibody have binding ability, the O.D value would change with AFB1 level.<br>Horizontal axis means Aflatoxin B1 concentration. Vertical axis means value of O.D 450 nm.</small></small></p><br>
                            <img src="http://2017.igem.org/wiki/images/1/1c/Csmuxnchu_result_fig12.png" style="width:100%">
                           <p><small><small><strong>Fig. 12:</strong>ELISA assay test 4000 times dilution of the ScFv fusion protein. In this time we desalted and increased the protein concentration to reduce interference. However, the result does not match with assumption. Horizontal axis means Aflatoxin B1 concentration. Vertical axis means value of O.D 450 nm.</small></small></p><br>


                      </div>
                   </ul>
                 </div>
                 </div>

            </div>
<a name="kit"></a>
        <div id="result-kit" class="m-block" >
        <img src="http://2017.igem.org/wiki/images/a/a8/T--CSMU_NCHU_Taiwan--safety-line.png" style="width:100%;" alt="">
<br>
          <img class = "center" src="http://2017.igem.org/wiki/images/f/fe/T--CSMU_NCHU_Taiwan--design-kits.png" alt="" style="width:40% ;margin-top:20px;">
           <h2>Design results</h2>
               <div class="pdf-area">
                      <span class="pdfbtn" id="kit-btn-1">Appearance & Manual & Interpretation<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="kit-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <p><strong>Appearance</strong></p>
                         <p>&emsp;Our kit design finally realized via 3D printing technology, as shown in the <strong>Fig. 13A</strong> to <strong>13C</strong>.</p>
                         <p>&emsp;The blue parts are the lid and grinder, which would be removed after grinding. The solvent part would later be added to mix with the food powder in the mixing tank. After food powder and solvent are well mixed, users may insert the test strip to the window beside the mixing tank, and start immunochromatography.  We used the traditional immunostrip here to test our kit, and it works well.</p>
                         <p>&emsp;Compare with traditional disposable kit, our kit features lightweight, reusability and recyclability. Every component are washable and reusable after used, and the PLA material is biodegradable, making it an eco-friendly product. </p><br>


                           <img src="http://2017.igem.org/wiki/images/1/16/Csmuxnchu_result_kit1.png" style="width:100%">
<img src="http://2017.igem.org/wiki/images/2/2a/Csmuxnchu_result_kit2.jpg" style="width:100%">
<img src="http://2017.igem.org/wiki/images/6/63/Csmuxnchu_result_kit3.jpg" style="width:100%">
                           <p><small><small><strong>Fig. 13:</strong>Appearance of our specific kit for test strip. (A) The kit without the lid. (B) The kit with the lid and grinder. (C)The kit with the solvent part.</small></small></p><br>
                          <p><strong>User manual</strong></p>
                          <p>To detect aflatoxin food, follow the steps bellow:</p>
                          <p>1.&emsp;Install the grinder on the top of the mixing tank, and then rotate it clockwise.</p>
                          <p>2.&emsp;Put 5 gram (approx.) of food into the grinder, and then cover the lid.</p>
                          <p>3.&emsp;Rotate the lid clockwise to grind the food until they are all grounded into powder. The food powder would fall into the mixing tank.</p>
                          <p>4.&emsp;Remove the grinder from the mixing tank by rotating the grinder counterclockwise.</p>
                          <p>5.&emsp;Install the solvent container on the mixing tank. Rotate the solvent container clockwise, then the bottom membrane of the solvent container would be punctured by the mixing tank. The solvent inside the solvent container would eventually flow into the mixing tank.</p>
                          <p>6.&emsp;Gently shake the kit and wait for 10 minutes.</p>
                          <p>7.&emsp;Insert the test strip to the side of the mixing tank, and make sure that the height of the fluid column climbs to the top. </p>
                          <p>8.&emsp;Read the result of the test strip.</p>


                             <p><strong>Result interpretation</strong></p>
                            <p>&emsp;The test strip provided in the kit is easy to read and identify. There are two red lines on the strip. The upper line is Control Line, and the bottom one is Test Line.</p>
                           <p>Reading the form in <strong>Table 3</strong> to understand the interpretation of the lines on the test strip：</p><br>
                            <img src="http://2017.igem.org/wiki/images/d/d9/Csmuxnchu_result_kit_1.png" style="width:100%" alt="">
<p><small><small><b>Table 3:</b>The different results of test
strip and their meaning.</small></small></p><br>
<img src="http://2017.igem.org/wiki/images/d/dd/T--CSMU_NCHU_Taiwan--StripResult.png" style="width:50%" alt="">
<p><small><small><b>Fig. 14:</b>The appearance of the results of test strip.</small></small></p><br>
                      </div>
                   </ul>
                 </div>
                 </div>
         </div>

               <img src="http://2017.igem.org/wiki/images/a/a8/T--CSMU_NCHU_Taiwan--safety-line.png" style="width:100%;" alt="">
                    <div id="result-reference" class="m-block" >
                              <h2>References</h2>
                             <div class="pdf-area">
                      <span class="pdfbtn" id="reference-btn-1">References<i class="fa fa-caret-down" aria-hidden="true">
</i></span>
                      <div class="ul-container" id="reference-1">
                         <ul class="ul-brown">
                          <div class="text-area">
                         <ul2 class="ul-style-num">
                              <li>1. Mersch-Sundermann, V., et al., Use of a human-derived liver cell line for the
detection of cytoprotective, antigenotoxic and cogenotoxic agents. Toxicology,
2004. <b>198</b>(1): p. 329-340.</li>
                               <li>2. Mah, L., A. El-Osta, and T. Karagiannis, γH2AX: a sensitive molecular marker of DNA
damage and repair. Leukemia, 2010. <b>24</b>(4): p. 679-686.</li>
                               <li>3. Hirao, A., et al., DNA Damage-Induced Activation of p53 by the Checkpoint Kinase
Chk2. Science, 2000. <b>287</b>(5459): p. 1824.</li>
                               <li>4. Shieh, S.-Y., et al., The human homologs of checkpoint kinases Chk1 and Cds1 (Chk2)
phosphorylate p53 at multiple DNA damage-inducible sites. Genes &amp; development,
2000. <b>14</b>(3): p. 289-300.</li>

                               <li>5. Kohn, K.W., Molecular interaction map of the mammalian cell cycle control and DNA
repair systems. Mol Biol Cell, 1999. <b>10</b>(8): p. 2703-34.</li>

                                <li>6. Taylor, M.C., et al., Identification and characterization of two families of
F420H2‐dependent reductases from Mycobacteria that catalyse aflatoxin
degradation. Molecular microbiology, 2010. <b>78</b>(3): p. 561-575.</li>

                          </ul2>



                      </div>
                   </ul>
                 </div>
                 </div>




                     </div>







          </div>
        </div>
      </div>

      <div class="path-tags">
        <ul>
          <a href="">
            <span>Antidote</span>
            <img src="http://2017.igem.org/wiki/images/a/ab/Csmuxnchu_result_flag.png" style="position: fixed; top: 190px;left:15%; width: 202px; height:40px;">
          </a>
          <li id="intro-btn" class="tag-btn">- Expression results</li>
          <li id="protein-btn" class="tag-btn">- Aflatoxin damage results</li>
          <li id="docking-btn" class="tag-btn">- Enzyme function results</li>
          <li id="conclusion-btn" class="tag-btn">- Cloning results</li>

          <br>
          <a href="http://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results#strip"><p style="font-size:18px;">Test strip</p></a>
           <a href="http://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results#kit"><p style="font-size:18px;">Device & APP</p></a>
           <a href="http://2017.igem.org/Team:CSMU_NCHU_Taiwan/Demonstrate"><p style="font-size:18px;">Demonstrate</p></a>


        </ul>
      </div>
    </div>

    <div class="top">
      <img src="http://2017.igem.org/wiki/images/5/52/T--CSMU_NCHU_Taiwan--top.png" alt="">
    </div>
  </body>
  <script type="text/javascript">
    $(document).ready(function(){
      $('.top').on('click', function(){
        $('html, body').animate({scrollTop: '0px'}, 500);
      });

      $("ul>li#intro-btn").click(function() {
        $('html, body').animate({
            scrollTop: $("#model-intro").offset().top
        }, 500);
      });
      $("#protein-btn").click(function() {
        $('html, body').animate({
            scrollTop: $("#model-protein").offset().top
        }, 500);
      });
      $("#docking-btn").click(function() {
        $('html, body').animate({
            scrollTop: $("#model-docking").offset().top
        }, 500);
      });
      $("#conclusion-btn").click(function() {
        $('html, body').animate({
            scrollTop: $("#model-conslusion").offset().top
        }, 500);
      });

  });
  $(document).ready(function(){

      for( var i = 1; i < 4; i++){
        $('#result-btn-' + i).click( toggleContainer(i) );
      }
       for( var j = 1; j < 4; j++){
        $('#damage-btn-' + j).click( toggleContainer2(j) );
      }
    for( var z = 1; z < 3; z++){
        $('#function-btn-' + z).click( toggleContainer3(z) );
      }
 for( var k = 1; k < 2; k++){
        $('#cloning-btn-' + k).click( toggleContainer4(k) );
      }
     $('#strip-btn-1').click( toggleContainer5() );
     $('#kit-btn-1').click( toggleContainer6() );
    $('#reference-btn-1').click( toggleContainer7() );
      $('.top').on('click', function(){
      $('html, body').animate({scrollTop: '0px'}, 500);
    });

});

function toggleContainer(i){
   return function(){
      if($('#result-'+i).is(":visible")){
        $('#result-'+i).slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#result-'+i).slideToggle(500);
      }
      $('.hp-bg-container').css('max-height','none')
      $('.main-content').css('min-height','200vh')
  }
}
function toggleContainer2(i){
   return function(){
      if($('#damage-'+i).is(":visible")){
        $('#damage-'+i).slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#damage-'+i).slideToggle(500);
      }

  }
}

function toggleContainer3(i){
   return function(){
      if($('#function-'+i).is(":visible")){
        $('#function-'+i).slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#function-'+i).slideToggle(500);
      }

  }
}
function toggleContainer4(i){
   return function(){
      if($('#cloning-'+i).is(":visible")){
        $('#cloning-'+i).slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#cloning-'+i).slideToggle(500);
      }

  }
}
function toggleContainer5(){
   return function(){
      if($('#strip-1').is(":visible")){
        $('#strip-1').slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#strip-1').slideToggle(500);
      }

  }
}
function toggleContainer6(){
   return function(){
      if($('#kit-1').is(":visible")){
        $('#kit-1').slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#kit-1').slideToggle(500);
      }

  }
}
function toggleContainer7(){
   return function(){
      if($('#reference-1').is(":visible")){
        $('#reference-1').slideToggle(500);
      }
      else{
        $('.ul-container:visible').slideToggle(500);
        $('#reference-1').slideToggle(500);
      }

  }
}
  </script>


</html>
